scripts/render_notebook_viz.py

```
@@ -0,0 +1,290 @@
+        kwargs.pop("_debug_overlays", None)
+        return original_graph_visualize(self, *args, **kwargs)
+
+    Graph.visualize = graph_visualize_wrapper  # type: ignore[assignment]
+
+    # Also patch the widget module reference (Graph.visualize imports from hypergraph.viz)
+    try:
+        import hypergraph.viz.widget as widget
+
+        widget.visualize = capture.visualize  # type: ignore[assignment]
+    except Exception:
+        pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)